src/utils/gitUtils.ts

```diff
@@ -165,6 +165,20 @@ export async function promptForDefaultBranch(context: IActionContext, repo: Repo
     }
 }
 
+export async function gitPull(repo: Repository): Promise<void> {
+    try {
+        const options: ProgressOptions = {
+            location: ProgressLocation.Notification,
+            title: localize('executingGitPull', 'Executing "git pull"...')
```

Comment:
Let's discuss offline, but I think we need a better messaging as to _why_ we're doing a git pull for the user.  Maybe something along the lines of `Performing "git pull" to retrieve xxx_workflow_file.yml...`

Reply:
@fiveisprime Thoughts